obsei/analyzer/ner_analyzer.py

```diff
@@ -77,11 +77,11 @@ def analyze_input(
         **kwargs,
     ) -> List[TextPayload]:
         analyzer_output: List[TextPayload] = []
+        source_response_list = TextSplitter().preprocess_input(
+            source_response_list, config=TextSplitterConfig(max_split_length=512)
```

Comment:
max_split_length should be configurable ideally default value should be based on model max length